### ISSUE TITLE
HTTP(s) proxy

### ISSUE THREAD

**Issue by flatrick** [flatrick/troubleshooting#4]
This can be crucial when you're troubleshooting a application that does secure HTTPS connections to other services. Without a tool like Charles Proxy or Fiddler, the data being sent between the application and the server will be encrypted and you'll most certainly not have the correct certificate to be able to decrypt the content manually making debugging/troubleshooting nearly impossible.  

But with Charles Proxy or Fiddler, not only will we be able to see what data is being sent, but the responses will also be readable!  
In other words, this might very well be a must-have tool in every troubleshooters toolbox!
